lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -279,8 +283,10 @@ class FatMemManager : public OpSemMemManager {
                             unsigned byteSz, uint64_t align) override {
     return mkFatMem(
         m_main.storeIntToMem(_val, mkRawPtr(ptr), mkRawMem(mem), byteSz, align),
-        m_main.storeIntToMem(0, mkRawPtr(ptr), mkSlot0Mem(mem), byteSz, align),
-        m_main.storeIntToMem(0, mkRawPtr(ptr), mkSlot1Mem(mem), byteSz, align));
+        m_slot0.storeIntToMem(m_ctx.alu().si(0, g_slotBitWidth), mkRawPtr(ptr),
+                              mkSlot0Mem(mem), g_slotByteWidth, align),
```

Comment:
I suggest to replace `m_slot0.storeIntToMem...` with `mkSlot0Mem(mem)`.

When writing a thin integer into fat memory we have a choice of what to do with the shadow. 
Zero it out, initialize non-deterministically, or leave it unchanged. I think leaving it unchanged
is best from performance perspective. So that would be my first choice. We can turn it into a
command line option if needed.

Reply:
Ack.